modules/apps/object/object-service/src/main/java/com/liferay/object/internal/action/util/ObjectEntryVariablesUtil.java

```diff
@@ -283,6 +290,28 @@ private static String _getContentType(
 		return dtoConverter.getContentType();
 	}
 
+	private static Map<String, Object> _getDefaultVariables(
+		ObjectDefinition objectDefinition, Set<String> keys) {
+
+		List<ObjectField> objectFields =
+			ObjectFieldLocalServiceUtil.getObjectFields(
```

Comment:
@p-albuquerque can we avoid using the `Util` here (`ObjectFieldLocalServiceUtil`)?

Reply:
I use because I found, inside this file, another one use it:

https://github.com/liferay-objects/liferay-portal/pull/1688/files/eab4ce84b4bf4154871fc485a30daaad395879ff..32e5b74936a7c203137b52373d9ef71972c18347#diff-80cb5d9e06dc6f1e4f959dd37ba1182f73a509c6d07d356df0ba8380bec16ad4L399-L401

Reply:
@p-albuquerque Ideally we should avoid this, but since there is another usage in the same class, let's try it this way